signing/MANIFEST.in

```diff
@@ -1,2 +1,17 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Can you confirm this was tested? (Hard to find docs to verify this file allows # comments.)